consoles/sshVirtsh.pm

```diff
@@ -34,7 +36,9 @@ sub new {
     $self->instance(get_var('VIRSH_INSTANCE', 1));
     # default name
     $self->name("openQA-SUT-" . $self->instance);
-    $self->_init_xml;
+    $self->vmm_family(get_var('VIRSH_VMM_FAMILY'));
+    $self->vmm_type(get_var('VIRSH_VMM_TYPE'));
```

Comment:
`VIRSH_VMM_FAMILY` and `VIRSH_VMM_TYPE` are meant to be mandatory. I don't think of KVM as a default one. Fixed.